### DISCUSSION TITLE
Support for Redis Data Types

### DISCUSSION THREAD

**Discussion by Eric24** [ArcadeData/arcadedb#1358] [General]
Since ArcadeDB supports Redis protocol, are there any plans to add support for some of the "unique" Redis data types, like Sets and Sorted Sets, and the SQL extensions to support the special functions that are possible with these data types, like SADD (only add to Set if value doesn't exist) and SPOP (remove a random element from a Set), etc.?

**Comment by lvca** [comment]
Hi @Eric24, we can improve the supported operators in the Redis handler. Please write down what you need and we'll devise a plan based on the priorities with other issues :-)

**Comment by lvca** [comment]
By the way, for those interested in contributing, the Redis protocol is simple and minimal. This is where everything happens:

https://github.com/ArcadeData/arcadedb/blob/5ba05710ae72abee2b1af4536fa20d77288dca32/redisw/src/main/java/com/arcadedb/redis/RedisNetworkExecutor.java#L115
